chainerx_cc/chainerx/routines/math.cc

```diff
@@ -554,6 +554,8 @@ Array LogSumExp(const Array& x, const OptionalAxes& axis, bool keepdims) {
 
 Array LogSoftmax(const Array& x, const OptionalAxes& axis) { return x - LogSumExp(x, axis.has_value() ? axis : OptionalAxes{1}, true); }
 
+Array Sigmoid(const Array& x) { return 1.0 / (1.0 + Exp(-x)); }
```

Comment:
Thanks for addressing the TODO about introducing Scalar / Array division, but would it for the sake of this PR be simpler to implement sigmoid as `Reciprocal(1.0 + Exp(-x))` (without introducing `operator/(Scalar, const Array&)`)? Also for the record, the first TODO should probably be addressed here instead https://github.com/chainer/chainer/blob/master/chainerx_cc/chainerx/routines/math.cc#L374